### PR TITLE
Support for kubernetes version 1.22

### PR DESCRIPTION
**Pull request by sancyx**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | fixes #96 
| License         | Apache 2.0

### What's in this PR?
Replace admissionregistration.k8s.io/v1beta1 not available from version 1.22 with admissionregistration.k8s.io/v1 available from version 1.16.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

**Review by pbalogh-sa — APPROVED** [review]

File: deploy/whitelist-crd.yaml [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
+    name: whitelistitems.security.banzaicloud.com
 spec:
-  group: security.banzaicloud.com
-  version: v1alpha1
-  names:
-    kind: WhiteListItem
-    plural: whitelistitems
-    singular: whitelistitem
-    shortnames:
-      - wl
```
Comment (pregnor) [inline deploy/whitelist-crd.yaml:13]: **Question**: why are we dropping `shortnames`?

Comment (sancyx) [reply deploy/whitelist-crd.yaml:13]: fixed

File: charts/anchore-policy-validator/templates/whitelist-crd.yaml [hunk 2]
```diff
@@ -1,35 +1,42 @@
...
   name: whitelistitems.security.banzaicloud.com
 spec:
   group: security.banzaicloud.com
-  version: v1alpha1
   names:
     kind: WhiteListItem
     plural: whitelistitems
     singular: whitelistitem
+    shortNames:
+      - wl
```
Comment (pregnor) [inline charts/anchore-policy-validator/templates/whitelist-crd.yaml:12]: **Question**: why has the `shortNames` been relocated from `deploy/whitelist-crd.yaml` to `charts/anchore-policy-validator/templates/whitelist-crd.yaml`?

Comment (pbalogh-sa) [reply charts/anchore-policy-validator/templates/whitelist-crd.yaml:12]: The deploy dir is used for testing purposes, while charts dir contains the hem chart. So the shortNames field was missing before in the case of the helm chart.

**Review by pregnor — APPROVED** [review]

**Pull request merged**